File: DropZone/DropZone.Core/Models/AAD/AADModel.cs
using System;

namespace DropZone.Core.Models
{
    public class AADModel
    {
        public long Id { get; set; }

        public DateTime? MaintenanceDate { get; set; }

        public DateTime ModifiedAt { get; set; }


        public long AADTypeId { get; set; }

        public long HashBlockId { get; set; }


        public AADTypeModel AADType { get; set; }

        public HashBlockModel HashBlock { get; set; }

        public ParachuteSystemModel ParachuteSystem { get; set; }
    }
}

File: DropZone/DropZone.Web/src/models/aads/aad.model.ts
import { AADTypeModel } from "./aadType.model";
import { ParachuteSystemModel } from "../parachuteSystems/parachuteSystem.model";
import { HashBlockModel } from "../hashBlocks/hashBlock.model";

export class AADModel {
	public Id: number;
	public MaintenanceDate: Date;
    public ModifiedAt: Date;

	public AADTypeId: number;
    public HashBlockId: number;
    
	public AADType: AADTypeModel;
	public HashBlock: HashBlockModel;
	public ParachuteSystem: ParachuteSystemModel;
};